package.json

```diff
@@ -491,12 +518,12 @@
         "test": "npm run test-compile && node ./out/src/tests/runTests.js"
     },
     "extensionDependencies": [
+        "vscode.git",
```

Comment:
I would recommend adding why this dependency is added. It is needed for draft workflow, adding them in the PR description would be nice for the others to know. Thank you.